internal/pkg/template/templates/workloads/partials/cf/subscribe.yml

```diff
@@ -108,6 +108,8 @@ DeadLetterPolicy:
           Resource: !GetAtt DeadLetterQueue.Arn
 {{- end}}{{- end}}
 
+{{- end }}{{/* endif .Subscribe */}}
```

Comment:
`{{/* endif .Subscribe */}}`!

Reply:
Is this the comment saying the `end` before is for `.Subscribe`? If so can we add for this https://github.com/aws/copilot-cli/blob/54269e26b714d4bf9d8aa67dc041152df35245a8/internal/pkg/template/templates/workloads/partials/cf/subscribe.yml#L229 as well?